Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -33,6 +33,8 @@ public class Server
 
         public bool NoDelay = true;
 
+        public int KeepAliveTimer;
```

Comment:
```suggestion
        public int KeepAliveTimer = 30;
```

Reply:
This is set at transport level. Do we really need to set this here considering the transport will override this?